### PR TITLE
Add unit tests for arcgis utils

### PR DESCRIPTION
**Pull request by jaumb**
closes https://github.com/codeforboston/BrooklineVoiceApp/issues/12

### CODE REVIEW

**Comment by jaumb** [comment]
@wesdrew will you please take a look at this one? I borrowed the technique from your crime incident unit test refactoring

**Comment by wesdrew** [comment]
Ah just saw this! I'll take a look soon

File: brooklinevoiceapp/mycity/test/test_constants.py [hunk 1]
```diff
@@ -144,4 +194,4 @@
         "GlobalID": "GlobalID"
     },
     "features": []
-}
\ No newline at end of file
+}
```
Comment (wesdrew) [inline brooklinevoiceapp/mycity/test/test_constants.py]: nitpick: no newline at end of file

File: brooklinevoiceapp/mycity/test/unit_tests/test_gis_utils.py [hunk 2]
```diff
@@ -0,0 +1,38 @@
+import mycity.test.test_constants as test_constants
+import mycity.utils.brookline_arcgis_api_utils as utils
+import mycity.test.unit_tests.base as base
+import unittest.mock as mock
+import copy
+
+
+SPACIAL = utils.SPATIAL_REFERENCE_PATH
+
```
Comment (wesdrew) [inline brooklinevoiceapp/mycity/test/unit_tests/test_gis_utils.py]: nitpick: only one newline after `SPACIAL`

File: brooklinevoiceapp/mycity/test/unit_tests/test_gis_utils.py [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+        expected_spacial = test_constants.SPACIAL_REFERENCE_MOCK
+        mock_requests = self.build_requests_stub(get_status=200,
+             get_data=test_constants.GET_ADDRESS_CANDIDATES_API_MOCK)
+        candidate_result, spacial_result = utils.get_first_address_candidate(self.test_address, mock_requests)
+        self.assertEqual(expected_candidate, candidate_result)
+        self.assertEqual(expected_spacial, spacial_result)
+
+    def test_geocode_address(self):
+        expected_result = copy.deepcopy(test_constants.LOCATION_MOCK)
+        expected_result[SPACIAL] = test_constants.SPACIAL_REFERENCE_MOCK
```
Comment (wesdrew) [inline brooklinevoiceapp/mycity/test/unit_tests/test_gis_utils.py]: Maybe move this building an `expected_result` by defining another constant in `test_constants.py`

**Comment by jaumb** [comment]
@wesdrew thanks, I made those changes, will you take another look when you have a chance?

**Comment by jaumb** [comment]
@jmartini will you please take a look at this one when you have a chance?

**Pull request merged**